src/column-layout/__tests__/column-layout.test.tsx

```diff
@@ -28,38 +33,48 @@ describe('ColumnLayout component', () => {
+          expect(wrapper.getElement()).not.toHaveClass(styles['grid-vertical-borders']);
+          expect(wrapper.getElement()).not.toHaveClass(styles['grid-horizontal-borders']);
+        });
 
-    it('applies horizontal styling', () => {
-      const wrapper = renderColumnLayout({ borders: 'horizontal' });
-      expect(wrapper.getElement()).not.toHaveClass(styles['grid-vertical-borders']);
-      expect(wrapper.getElement()).toHaveClass(styles['grid-horizontal-borders']);
-    });
+        it('applies vertical styling', () => {
+          const wrapper = renderColumnLayout({ ...moreProps, borders: 'vertical' });
+          expect(wrapper.getElement()).toHaveClass(styles['grid-vertical-borders']);
+          expect(wrapper.getElement()).not.toHaveClass(styles['grid-horizontal-borders']);
+        });
 
-    it('applies both horizontal and vertical when "all" is provided', () => {
-      const wrapper = renderColumnLayout({ borders: 'all' });
-      expect(wrapper.getElement()).toHaveClass(styles['grid-vertical-borders']);
-      expect(wrapper.getElement()).toHaveClass(styles['grid-horizontal-borders']);
-    });
-  });
+        it('applies horizontal styling', () => {
+          const wrapper = renderColumnLayout({ ...moreProps, borders: 'horizontal' });
+          expect(wrapper.getElement()).not.toHaveClass(styles['grid-vertical-borders']);
+          expect(wrapper.getElement()).toHaveClass(styles['grid-horizontal-borders']);
+        });
+
+        it('applies both horizontal and vertical when "all" is provided', () => {
+          const wrapper = renderColumnLayout({ ...moreProps, borders: 'all' });
+          expect(wrapper.getElement()).toHaveClass(styles['grid-vertical-borders']);
+          expect(wrapper.getElement()).toHaveClass(styles['grid-horizontal-borders']);
+        });
+      });
 
-  describe('text-grid variant', () => {
-    it('disables borders even when a value for borders is provided', () => {
-      ['none', 'vertical', 'horizontal', 'all'].forEach(borders => {
-        const wrapper = renderColumnLayout({ variant: 'text-grid', borders: borders as ColumnLayoutProps['borders'] });
-        expect(wrapper.getElement()).not.toHaveClass(styles['grid-vertical-borders']);
-        expect(wrapper.getElement()).not.toHaveClass(styles['grid-horizontal-borders']);
+      describe('text-grid variant', () => {
+        it('disables borders even when a value for borders is provided', () => {
+          ['none', 'vertical', 'horizontal', 'all'].forEach(borders => {
+            const wrapper = renderColumnLayout({
+              ...moreProps,
+              variant: 'text-grid',
+              borders: borders as ColumnLayoutProps['borders'],
+            });
+            expect(wrapper.getElement()).not.toHaveClass(styles['grid-vertical-borders']);
+            expect(wrapper.getElement()).not.toHaveClass(styles['grid-horizontal-borders']);
+          });
+        });
       });
     });
   });
```

Comment:
Can we add test for cssColumnlayout to check `columnLayoutColumnCount`?

Reply:
The main logic is tested with the `calculcateCssColumnCount` utility. But I'll add a test to make sure that the column value is properly transferred to the CSS Custom Property.